src/types/targetValidators.ts

```diff
@@ -81,6 +82,14 @@ export const validateAppLoc = (appLoc: string): string => {
   return appLoc
 }
 
+export const validateDocConfig = (docConfig: DocConfig): DocConfig => {
+  if (!docConfig) {
+    throw new Error('Invalid doc config: JSON cannot be null or undefined.')
+  }
```

Comment:
I think it **can** be undefined - it should be possible to generate with a very (very) minimal sasjsconfig.json file.

Check this one out:  https://github.com/sasjs/core/blob/main/sasjs/sasjsconfig.json

(and this one IS used to generate docs, eg the https://core.sasjs.io)